backend/apiConfig.ts

```diff
@@ -0,0 +1,31 @@
+import 'dotenv/config'
```

Comment:
This file is part of config.ts in the template project. I have separated this out to make the file changes simpler.